python/lsst/ts/electrometer/controller.py

```diff
@@ -0,0 +1,47 @@
+import enum
+
+
+class UnitMode(enum.IntEnum):
```

Comment:
Are these enums from the CSC interface? if so, they need to go to `ts_idl`

Reply:
No, the enums are for the controller specifically. The enums in csc need to go into ts_idl, I guess.